controllers/users.js

```diff
@@ -191,6 +192,30 @@ const getUsers = async (req, res) => {
       }
     }
 
+    const isDeparted = req.query.departed === "true";
+
+    if (isDeparted) {
+      if (!dev) {
+        return res.boom.notFound("Route not found");
+      }
+      try {
+        const result = await dataAccess.retrieveUsers({ query: req.query });
```

Comment:
What is it returning?

Reply:
An object
`{ users: [], nextId: '' , prevId: ''}`